src/components/vacation-requests-table/vacation-requests-table-toolbar/toolbar-form/toolbar-form-fields.tsx

```diff
@@ -55,7 +53,8 @@ const ToolbarFormFields = ({
       days: calculateTotalVacationDays(
         dateRange.start,
         dateRange.end,
-        getWorkingWeek(loggedInPerson)
+        // getWorkingWeek(loggedInUser)
+        [true, true, true, true, true, false, false]
```

Comment:
What if the user is part time? I realize it might be beyond the scope of this feature to resolve this issue, but think about how we will resolve it in the future. I'm guessing we will need some info from SeveraApi. It might be work also storing such information in our User.

Reply:
True, that is hard-coded at the moment for testing purposes, but we can discuss it later how to implement it. And yes, I left it to be solved in the next issue.